buildlib/azure-pipelines-release.yml

```diff
@@ -107,3 +107,22 @@ stages:
         parameters:
           arch: aarch64
           demands: ucx-arm64
+
+      - template: jucx/jucx-build.yml
+        parameters:
+          arch: amd64
+          container: centos7_cuda11_x86_64
+          demands: ucx_docker
+
+      - template: jucx/jucx-build.yml
+        parameters:
+          arch: aarch64
+          container: centos8_cuda11_aarch64
+          demands: ucx-arm64
+
+      - template: jucx/jucx-publish.yml
+        parameters:
+          ${{ if eq(variables['Build.Reason'], 'IndividualCI') }}:
+            target: publish-release
+          ${{ if eq(variables['Build.Reason'], 'PullRequest') }}:
+            target: package
```

Comment:
why needed - we also check Build.Reason as a condition for "Publish JUCX jar to maven central" task